posts/2015-05-24-es6-gotchas.md

```diff
@@ -46,6 +46,21 @@ export default class ComponentName extends React.Component {
   }
 ~~~
 
+Another alternative syntax is to use the `::` ES7 [bind](https://github.com/zenparsing/es-function-bind)
+
+~~~js
+export default class ComponentName extends React.Component {
+
+  handleClick(e) => {
+    ...
+  }
+  
+  render(){
+    return (
+      <button onClick={::this.handleClick}> click </button> 
```

Comment:
Also when you use it to listen to your stores:

```js
componentDidMount() {
  MyStore.listen(::this.onChange);
},

componentWillUnmount() {
  MyStore.unlisten(::this.onChange); // memory leak.
}
```

worth mentioning since `::` isn't a silver bullet you can blindly apply to everything.